lib/default.rc

```diff
@@ -2,8 +2,8 @@
 
 PROJECT_NAME=overleaf
 SERVER_PRO=false
-SHARELATEX_DATA_PATH=data/sharelatex
-SHARELATEX_PORT=80
+OVERLEAF_DATA_PATH=data/sharelatex
```

Comment:
cc @briangough WDYT about pointing new setups at `data/overleaf`?

Reply:
Oh, sorry, I meant to comment in the `lib/config-seed/overleaf.rc` :see_no_evil: 

The default.rc is used by all deployments as fallback and should not migrate that one.

Reply:
I'll update `OVERLEAF_DATA_PATH` to `data/overleaf` in `lib/config-seed/overleaf.rc` 👍🏻

Reply:
Done in https://github.com/overleaf/toolkit/pull/217/commits/4f848c3ee31f3cf1519a2ef5532d3e6c46de5706